recipes/pantax/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pantax" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/LuoGroup2023/PanTax/releases/download/v{{ version }}/{{ name }}.tar.gz  
+    sha256: c14a46b41f580cf39ad502049d267cd31bbfd4dc05028215cc669c0e52f58ce7
+
+  - url: https://files.pythonhosted.org/packages/07/0f/6039cf6e22f9cbec57cdedff103949e4856c18fd2cb714bc70cdc8b52941/gurobipy-11.0.2-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl 
+    sha256: 164e72462c2f1a705fcec73acc060aa639e745827bfe877714ff7b2ccabc5237  
+    folder: gurobi11  
+
+  - url: https://github.com/vgteam/vg/releases/download/v1.52.0/vg    
+    sha256: 07776b51aa6494b45982e4134eb84005ddc743e81c67e61fa52bff211d81777c 
+    folder: vg
+
+build:
+  skip: True  # [osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}   
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - python ==3.10
+    - pip
+  run:
+    - python =3.10
```

Comment:
We do not recommend to use those very strict pins. Are they needed?

Reply:
I hope this setting can match the version of my tool, which is beneficial for fully reproducing my running results.